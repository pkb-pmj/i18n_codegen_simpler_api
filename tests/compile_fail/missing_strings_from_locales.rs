use i18n_codegen::i18n;

i18n!("../../../../tests/compile_fail/missing_strings_from_locales");

fn main() {}
