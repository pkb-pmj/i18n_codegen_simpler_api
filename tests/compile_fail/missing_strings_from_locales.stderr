error: proc macro panicked
 --> tests/compile_fail/missing_strings_from_locales.rs:3:1
  |
3 | i18n!("../../../../tests/compile_fail/missing_strings_from_locales");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Some locales are missing keys
          `Da` is missing:
            `bar`

          `En` is missing:
            `foo`
